sections/futures/MarketDetails/MarketDetails.tsx

```diff
@@ -190,24 +200,47 @@ const MarketSkew: React.FC<MarketDetailsProps> = memo(({ mobile }) => {
 	);
 });
 
-const OpenInterestShortDetail = memo(() => {
+const OpenInterestDetail: React.FC<OpenInterestDetailProps> = memo(({ mobile, isLong }) => {
 	const marketInfo = useAppSelector(selectMarketInfo);
 	const oiCap = marketInfo?.marketLimitUsd
 		? formatDollars(marketInfo?.marketLimitUsd, { truncate: true })
 		: null;
+	const openInterestType = isLong ? 'longUSD' : 'shortUSD';
+	const formattedUSD = marketInfo?.openInterest[openInterestType]
+		? formatDollars(marketInfo?.openInterest[openInterestType], { truncate: true })
+		: NO_VALUE;
+
+	const mobileValue = (
+		<FlexDivCol>
+			<div>{formattedUSD}</div>
+			<Body mono size="small" color="secondary" weight="bold">
+				{oiCap}
+			</Body>
+		</FlexDivCol>
+	);
+
+	const desktopValue = `${formattedUSD}/${oiCap}`;
+
+	const dataKey = `openInterest${isLong ? 'Long' : 'Short'}${
+		mobile ? 'Mobile' : ''
+	}` as keyof typeof MarketDataKey;
 
 	return (
 		<MarketDetail
-			dataKey={MarketDataKey.openInterestShort}
-			value={
-				marketInfo?.openInterest.shortUSD
-					? `${formatDollars(marketInfo?.openInterest.shortUSD, { truncate: true })}/${oiCap}`
-					: NO_VALUE
-			}
+			mobile={mobile}
+			dataKey={MarketDataKey[dataKey]}
+			value={mobile ? mobileValue : desktopValue}
 		/>
 	);
 });
 
+const OpenInterestLongDetail: React.FC<MarketDetailsProps> = (props) => (
+	<OpenInterestDetail isLong {...props} />
```

Comment:
Seems a bit overkill to have two separate components for a hardcoded prop, should we just pass the isLong prop at the point these are rendered instead, so we only use a OpenInterestDetail component?

Reply:
Good idea. Thanks.